andela/ah-titans
# #160654507 Social auth verifcation

#### What does this PR do?
Verifies when user logs in with social media account.

#### Description of tasks to be completed
User should be able to be verified if they log in with social media without being sent an email to do so.

#### How should this be manually tested?
```sh
$ git clone https://github.com/andela/ah-titans.git
```
```sh
$ git checkout -b ch-social-auth-verify-modify-160654507
```
```sh
$ cd ah-titans/authors
```

> Add **.env** file and copy **example.env** content.
> Add your local database configuration settings to your **.env** file

```sh
$ source .env
```
```sh
$ python3.6 -m venv venv
```
```sh
$ source venv/bin/activate
```
```sh
$ pip install -r requirements.txt
```
```sh
$ python manage.py makemigrations
```
```sh
$ python manage.py runserver
```

#### Any background context you want to provide?
No

#### What are the relevant Pivotal Tracker Stories?
[#160654507](https://www.pivotaltracker.com/story/show/160654507)
